addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -3,6 +3,26 @@
 
     <t t-name="web.ControlPanel">
         <div class="o_control_panel d-flex flex-column gap-3 gap-lg-1 px-3 pt-2 pb-3" t-ref="root" data-command-category="actions">
+            <div class="o_topbar_actions d-flex w-100 align-items-center justify-content-center gap-2 mb-2" t-att-class="{visible: state.topbarInfos.showTopbar}">
```

Comment:
Why not using a `t-if` if the topbar should not be displayed? 🧐 

No need to have it in the dom if we are sure to not display it, right?

```suggestion
            <div t-if="state.topbarInfos.showTopbar" class="o_topbar_actions d-flex w-100 align-items-center justify-content-center gap-2 mb-2">
```